tools/Google.Cloud.Tools.VersionCompat/Program.cs

```diff
@@ -93,12 +93,26 @@ private enum ExitCodeBehavior
             MajorAndMinor = 2,
         }
 
+        private enum OutputBehavior
```

Comment:
Possibly make this a flags enum, then you can conditionalize it more easily?

Reply:
done